docs/content/topics/scripting/guide.md

```diff
@@ -9,1401 +9,867 @@ aliases:
-Creating secret goodbye-1504057229149-master
-Creating Job Cache PVC github-com-brigadecore-empty-testbed-hello
-undefined
-Creating Job Cache PVC github-com-brigadecore-empty-testbed-goodbye
-undefined
-Creating pod hello-1504057229136-master
-Creating pod goodbye-1504057229149-master
-Timeout set at 900000
-Timeout set at 900000
-default/hello-1504057229136-master phase Pending
-default/goodbye-1504057229149-master phase Pending
-default/goodbye-1504057229149-master phase Pending
-default/hello-1504057229136-master phase Pending
-default/goodbye-1504057229149-master phase Pending
-default/hello-1504057229136-master phase Pending
-default/goodbye-1504057229149-master phase Pending
-default/hello-1504057229136-master phase Pending
-default/goodbye-1504057229149-master phase Pending
-default/hello-1504057229136-master phase Pending
-default/hello-1504057229136-master phase Pending
-default/goodbye-1504057229149-master phase Pending
-default/hello-1504057229136-master phase Succeeded
-default/goodbye-1504057229149-master phase Succeeded
-Destroying PVC named brigade-worker-01brx9n20bsjxeweggtzb7fpka-master
-Done in 15.17s.
-```
+const { events, Job } = require("@brigadecore/brigadier");
 
-What is surprising is that if we look at the output above, we see that both jobs seem to
-be running at the same time. This is because when we start a job in Brigade, it runs
-asynchronously. Another way to phrase that is that jobs run _parallel by default_.
+events.on("brigade.sh/cli", "exec", async event => {
+  let job1 = new Job("my-first-job", "debian", event);
+  job1.primaryContainer.command = ["echo"];
+  job1.primaryContainer.arguments = ["My first job!"];
+  await job1.run();
 
-Again, if you want to view the output of each job, you can use the `kubectl logs` command
-for each pod.
+  let job2 = new Job("my-second-job", "debian", event);
+  job2.primaryContainer.command = ["echo"];
+  job2.primaryContainer.arguments = ["My second job!"];
+  await job2.run();
+});
 
-If we want these two pods to run _in order_, with `hello` running to completion before
-`goodbye` starts, we can do that by creating what in JavaScript is called a "Promise
-chain":
-
-```javascript
-const { events, Job } = require("brigadier")
-
-events.on("exec", () => {
-  var hello = new Job("hello", "alpine:3.4")
-  hello.tasks = [
-    "echo Hello",
-    "echo World"
-  ]
-
-  var goodbye = new Job("goodbye", "alpine:3.4")
-  goodbye.tasks = [
-    "echo Goodbye",
-    "echo World"
-  ]
-
-  hello.run().then(() => {
-    goodbye.run()
-  })
-})
+events.process();
 ```
-[brigade-08.js](../../examples/brigade-08.js)
+[04-simple-pipeline](https://github.com/brigadecore/brigade/tree/v2/examples/04-simple-pipeline)
 
-The important new part is at the end. We have replaced this:
-
-```javascript
-hello.run()
-goodbye.run()
-```
+In this example we create two jobs (`my-first-job` and `my-second-job`). Each
+starts a Debian container and prints a message, then exits.
```

Comment:
I think we should call out that the `await` keyword means job2 won't start until job1 completes, so these jobs implicitly execute in serial.